contracts/UFragmentsPolicy.sol

```diff
@@ -182,6 +182,31 @@ contract UFragmentsPolicy is Ownable {
         rebaseLag = rebaseLag_;
     }
 
+    /**
+     * @notice Sets the hyper-parameters which control the timing and frequency of
+     *         rebase operations.
+     *         a) the minimum time period that must elapse between rebase cycles.
+     *         b) the rebase window offset parameter.
+     *         c) the rebase window length parameter.
+     * @param minRebaseTimeIntervalSec_ More than this much time must pass between rebase
+     *        rebase operations, in seconds.
+     * @param rebaseWindowOffsetSec_ The number of seconds from midnight when
+     *        the rebase window begins.
+     * @param rebaseWindowLengthSec_ The size of the rebase window in seconds.
+     */
+    function setRebaseTimingParameters(uint256 minRebaseTimeIntervalSec_,
```

Comment:
Let's follow the formatting guide here: https://solidity.readthedocs.io/en/v0.5.9/style-guide.html#maximum-line-length

Reply:
👍